todo_app/app.py

```diff
@@ -8,4 +9,14 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items=get_items()
+    return render_template('index.html',items=items)
+
+@app.route('/add-todo',methods=["POST"])
+def add_todo():
+    new_todolist=request.form.get('title')
+    add_item(new_todolist)
+    return redirect("/")
+
+
+  
```

Comment:
Really minor but everything else looks fantastic - fussy dev teams will often insist on particular patterns in the code, including whitespace/line break expectations, to make sure things are standard and to make the review process as simple as possible by removing unnecessary changes - these bonus extra lines here are a nice example of something we might start to care about once we're happy with the logic!
```suggestion
```